diff --git a/plugin/config.vim b/plugin/config.vim
index f3fcf9d..e2df124 100644
--- a/plugin/config.vim
+++ b/plugin/config.vim
@@ -50,6 +50,8 @@ endif
 
         let name = s:plug_note_getname("Plug")
         if empty(name) | let name = s:plug_note_getname("Note") | endif
+
+        " Forwardto K's handler
         if empty(name) | call feedkeys("K", 'n') | return | endif
 
         "if has_key(g:plugs, name)
